E3Next/Processors/Basics.cs

```diff
@@ -695,10 +695,13 @@ public static void CheckManaResources()
 
                 bool hasManaStone = MQ.Query<bool>("${Bool[${FindItem[=Manastone]}]}");
                 bool amIStanding = MQ.Query<bool>("${Me.Standing}");
-                if (MQ.Query<bool>("${Me.Invis}")) return;
+               
 
                 if (hasManaStone && amIStanding)
                 {
+                    MQ.Delay(0);
```

Comment:
what does a delay(0) even do?

Reply:
gives control back to MQ/EQ, so a new message can be pushed into the queues.

Reply:
I thought about making it a method just to be more descriptive.

Reply:
I think that makes sense; something like YieldToEq() or something